src/plays/url-shortner/UrlShortner.js

```diff
@@ -0,0 +1,64 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import PlayHeader from 'common/playlists/PlayHeader';
+import { useState } from 'react';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import axios from 'axios';
+
+function UrlShortner(props) {
+  // Your Code Start below.
+  const [userInput, setUserInput] = useState('');
+  const [shortenedLink, setShortenedLink] = useState('');
+
+  const handleSubmit = async () => {
+    try {
+      const response = await axios(`https://api.shrtco.de/v2/shorten?url=${userInput}`);
+      setShortenedLink(response.data.result.full_short_link);
+    } catch (e) {
+      // eslint-disable-next-line no-console
```

Comment:
if the the userInput is not a valid url API gives error handle it in the catch block

Reply:
Error handling is done.